dockers/docker-base/Dockerfile.j2

```diff
@@ -33,9 +33,16 @@ RUN apt-get -y purge                    \
     exim4-config                        \
     exim4-daemon-light
 
-## Clean up apt
-## Remove /var/lib/apt/lists/*, could be obsoleted for derived images
-RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y;      \
-    rm -rf /var/lib/apt/lists/*;                                        \
-    rm -rf /tmp/*;
+{% if docker_base_dbgs %}
+## Install common debug-packages
+RUN apt-get -y install \
+{% for dbg_pkg in docker_base_dbgs.split(' ') -%}
+    {{ dbg_pkg }}{{' '}}
+{%- endfor %}
+{% endif %}
 
+## Clean up
+RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
+## Note: NO removing /var/lib/apt/lists/*, shared by all derived images
```

Comment:
we should remove, the derived images will do apt-get update again. We can save the baseimage iamge if we remove.